code/modules/surgery/organs/organ_external.dm

```diff
@@ -196,6 +199,10 @@
 
 	if(status & ORGAN_BROKEN && prob(40) && brute)
 		owner.emote("scream")	//getting hit on broken hand hurts
+	if(status & ORGAN_SPLINTED && prob((brute + burn)*4)) //taking damage to splinted limbs removes the splints
+		status &= ~ORGAN_SPLINTED
+		owner.visible_message("<span class=danger>The splint on [owner]'s left arm unravels from their [src.name]!</span>","<span class=userdanger>The splint on your [src.name] unravels!")
```

Comment:
Your spans are broken... neither have the CSS type in quotes, and the second span doesn't get closed.